### PR TITLE
Slight refactor towards preserving order in which users where added to the thread

### PR DESCRIPTION
**Pull request by stkem**

### CODE REVIEW

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala [hunk 1]
```diff
@@ -398,7 +398,7 @@ class MessagingController @Inject() (
     "6f21b520-87e7-4053-9676-85762e96970a"  // jenny
   )
 
-  def constructRecipientSet(userExtIds: Seq[ExternalId[User]]) : Future[Set[Id[User]]] = {
+  def constructRecipientSet(userExtIds: Seq[ExternalId[User]]) : Future[Seq[Id[User]]] = {
```
Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala:401]: might consider renaming the method

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala [hunk 2]
```diff
@@ -410,12 +410,12 @@ class MessagingController @Inject() (
         case notAGroup => Seq(notAGroup)
       }
     }
-    shoebox.getUserIdsByExternalIds(loadedUser.flatten).map(_.toSet)
+    shoebox.getUserIdsByExternalIds(loadedUser.flatten)
```
Comment (2is10) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala:413]: looks like this call doesn’t necessarily preserve order

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala (hunk 1 above)
Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala:401]: ah, good point. missed that one

File: kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala (hunk 2 above)
Comment (stkem) [inline kifi-backend/modules/eliza/app/com/keepit/eliza/MessagingController.scala:413]: fixed

**Pull request merged**